src/box/txn_limbo.c

```diff
@@ -268,6 +267,21 @@ txn_limbo_assign_lsn(struct txn_limbo *limbo, struct txn_limbo_entry *entry,
 static void
 txn_limbo_write_rollback(struct txn_limbo *limbo, int64_t lsn);
 
+void
+txn_limbo_update_confirmed_entry_after_rollback(struct txn_limbo *limbo,
+						int64_t lsn)
+{
+	struct txn_limbo_entry *last_entry = rlist_last_entry(
+		limbo->queue, struct txn_limbo_entry, in_queue);
+	if (limbo->confirmed_entry == last_entry) {
+		limbo->confirmed_entry_is_valid = false;
+	} else if (lsn <= rlist_next_entry(limbo->confirmed_entry,
+					   in_queue)->lsn) {
+		txn_limbo.confirmed_entry = last_entry;
```

Comment:
`txn_limbo` is the global object, you probably wanted to use `limbo->`, from the arguments of this function.

Reply:
Fixed.